TESTING.md

```diff
@@ -0,0 +1,114 @@
+<p align="center">
```

Comment:
@bindoon Please review this documentation to see if is clear enough to get maintainers started.